spk/python/Makefile

```diff
@@ -13,7 +13,7 @@ DEPENDS += cross/sqlalchemy cross/pyaudio cross/pyalsa cross/lxml cross/twisted
 DEPENDS += cross/zope.interface cross/markupsafe cross/sphinxbase cross/pocketsphinx
 DEPENDS += cross/msgpack-python cross/psutil cross/pymongo cross/pyyaml cross/uwsgi
 
-WHEELS = psutil uwsgi
+WHEELS = psutil uwsgi msgpack-python
```

Comment:
Almost ;) Could you put these in alphabetical order please? (Don't bother with the DEPENDS here though, that could use a cleanup)